internal/ethapi/api.go

```diff
@@ -643,6 +643,48 @@ func (s *PublicBlockChainAPI) BlockNumber() hexutil.Uint64 {
 	return hexutil.Uint64(header.Number.Uint64())
 }
 
+// GetBlockReceipts returns all the transaction receipts for the given block hash.
+func (s *PublicBlockChainAPI) GetReceipts(ctx context.Context, blockHash common.Hash) ([]map[string]interface{}, error) {
```

Comment:
For clarity, I would recommend changing the function name "GetReceipts" to "GetReceiptsByBlockHash".

Reply:
Thank you for feedback. 
Conversely, This naming is for maintaining clarity.
There are "GetReceipts" functions which provide exactly same result(ex. eth/api_backend.go) for internal usage. 

I want to hear your opinion about this. Thank you :)

Reply:
In case of "eth_getReceipts", it is an externally exposed API, so like other APIs (ex eth_getBlockByHash, eth_getTransactionByHash, etc) it seems necessary to make it clear which parameters are used.